_posts/2021-04-29-foreman-community-newsletter-april-2021.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: Foreman Community Newsletter (April 2021)
+date: 2021-04-29 17:48:21
+author: Melanie Corr
+tags:
+- foreman
+---
+
+Katello hit a major milestone, automate your way into summer, host details page updates, and much more!
+
+<!--more-->
+
+
+## At long last, Katello 4.0 has arrived!
+
+This month, the Katello community reached a major milestone with the release of Katello 4.0. This release involved years of work, both on the [Pulp](https://pulpproject.org/) side, with the complete architectural overhaul that [led to the release of Pulp 3](https://opensource.com/article/20/8/manage-repositories-pulp), as well as massive changes on the Katello side, which were demoed at regular intervals over the last year or more. As well as this, a massive thank you is owed to everyone who asked questions and reported issues on the buildup to this major release.
+
+As well as the removal of Pulp 2, here are some of the major highlights of this release:
+
+* Katello repositories are now hosted at [yum.theforeman.org](yum.theforeman.org).
+* New Content View page user interface updates.
+* Content View Version added to Reporting Engine Template.
+* Number of Red Hat subscriptions consumed by host added to Entitlements report.
+* Added option to select Operating System in Register Content Host.
+
+For more information, check out the [release announcement](https://community.theforeman.org/t/katello-4-0-is-released/23245?u=mcorr).
+
+## Introduction to Katello
+
+On the topic of Katello, [@iballou](https://community.theforeman.org/u/iballou/summary) gave an introductory overview of Katello and its main features and workflows. You can rewatch the event on YouTube and if you have any questions, please post to our [Community forum](https://community.theforeman.org/).
```

Comment:
Link to the YouTube video would be cool?

Edit: oh, it's embedded below, maybe reword to mention that fact? ;)